recipe/build.sh

```diff
@@ -8,8 +8,6 @@
 # disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
 #
 # *****************************************************************
-export CUDA_HOME=$PREFIX
```

Comment:
This shouldn't actually be removed. When we do this, we will be skipping over the nvcc package. I think instead it should be `CUDA_HOME=$CONDA_PREFIX`